### ISSUE TITLE
Filter for specific topics on Interview Questions

### ISSUE THREAD

**Issue by DamianRubio** [petrovicstefanrs/30_seconds_of_knowledge#7]
Nowadays, the extension allows you to filter out questions on topics like JS, Python or React and also for Interview Questions. But it does not discard questions on those topics to appear as an Interview Question.

For example, if someone is interested in learning Python (:raising_hand_man:)  and Interview Questions about Python, even though she discards JS questions some of them may appear on the 'Interview Questions' section.

It would be lovely to have this option.

EDIT: By the way, I'm new to opening issues on other peoples repos and I'm not able to find the option to label the issue as an enhancement. Don't even know if I should be allowed to do it.

**Comment by petrovicstefanrs** [comment]
Ohh that could prove to be a bit difficult, since all the snippets for interviews are written by community and grouped together without any separation. And mostly they focus on frontend questions in JS as you noticed. If in the future i find some collaborators that would be willing to write interview questions for different technologies I would be able to separate them.

TL;DR Sorry, but currently not possible, however would like to do it as soon there is a chance for it.

**Comment by mc-suchecki** [comment]
I second this. It seems like this extension would get more technologies/languages supported. Because of this, I suspect that this issue would become more and more important over time. 😉

BTW - thanks for the extension, I love it!
